content/acra/security-controls/client-id-sql-detection/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: ClientID SQL Detection
+weight: 12
+---
+
+## ClientID Detection from SQL
+
+{{< hint info >}}
+This feature is available in [Acra Enterprise Edition](/acra/enterprise-edition/) only.
+{{< /hint>}}
+
```

Comment:
we should add 1-2 sentence on why this feature is needed and why users will want to use it. I.e. some use cases would be useful.